mender-convert-modify

```diff
@@ -225,19 +225,34 @@ EOF
 log_info "Installing Mender client and related files"
 
 if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    MENDER_CLIENT_DEB_VERSION="$MENDER_CLIENT_VERSION"
+    if [ "$MENDER_CLIENT_VERSION" = "auto" ]; then
+        deb_distro=$(probe_debian_distro_name)
+        deb_codename=$(probe_debian_distro_codename)
+        if ([ "$deb_distro" = "debian" ] && [ "$deb_codename" = "bullseye" ]) \
+            || ([ "$deb_distro" = "debian" ] && [ "$deb_codename" = "buster" ]) \
+            || ([ "$deb_distro" = "ubuntu" ] && [ "$deb_codename" = "jammy" ]) \
+            || ([ "$deb_distro" = "ubuntu" ] && [ "$deb_codename" = "focal" ]); then
```

Comment:
Suggestion: Could also consider using `VERSION_ID` which is formatted as `12` and `22.04` for Debian and Ubuntu, respectively. I *think* you can use numeric comparisons on both, but not completely sure about the second one (unsure how the shell treats decimal point).